.github/workflows/download-link-check-issue.yml

```diff
@@ -0,0 +1,61 @@
+name: Scheduled DVC Download Link Check
+
+on:
+  workflow_call:
+    inputs:
+      owner:
+        required: false
+        type: string
+        default: "${{ github.event.repository.owner.login }}"
+      repo:
+        required: false
+        type: string
+        default: "${{ github.event.repository.name }}"
+
+jobs:
+  check_links_and_make_issue:
+    name: Check Download Links and Raise Issue
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Run Lychee
+        uses: lycheeverse/lychee-action@v1.0.8
+        with:
+          args:
+            --verbose https://dvc.org/ --base https://dvc.org/ --include
+            /download/
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+      - name: Find latest open Link Check issue
+        uses: actions/github-script@v6
+        id: set-result
+        env:
+          OWNER: ${{ inputs.owner }}
+          REPO: ${{ inputs.repo }}
+        with:
+          script: |
+            const { OWNER, REPO } = process.env
+            const options = {
+              owner: OWNER,
+              repo: REPO,
+              state: "open",
+              creator: "github-actions[bot]",
+              sort: "created"
+            }
+            const issues = await github.rest.issues.listForRepo(options)
+            const latestOpenLinkCheckIssue = issues.data.find(
+              (issue)=>(
+                issue.title === "DVC Download Link Checker Report"
+              )
+            )
+            if(latestOpenLinkCheckIssue) return latestLinkCheckIssue.number
```

Comment:
I couldn't find an Action for this specifically, so I used `github-script` to prototype.

`github-script` doesn't seem to natively support `workflow_call` inputs, so I used environment variables to smuggle them in.